docs/apm/troubleshooting.asciidoc

```diff
@@ -164,39 +141,35 @@ You can also use the Agent's public API to manually set a name for the transacti
 === Fields are not searchable
 
 In Elasticsearch, index templates are used to define settings and mappings that determine how fields should be analyzed.
-The recommended index template file for APM Server is installed by the APM Server packages.
-This template, by default, enables and disables indexing on certain fields.
+The recommended index templates for APM are installed by {fleet} when the Elastic APM integration is installed.
+These templates, by default, enable and disable indexing on certain fields.
 
 As an example, some agents store cookie values in `http.request.cookies`.
 Since `http.request` has disabled dynamic indexing, and `http.request.cookies` is not declared in a custom mapping,
 the values in `http.request.cookies` are not indexed and thus not searchable.
 
-*Ensure an index pattern exists*
-As a first step, you should ensure the correct index pattern exists.
-Open the main menu, then click *Stack Management > Index Patterns*.
-In the pattern list, you should see an apm index pattern; The default is `apm-*`.
-If you don't, the index pattern doesn't exist. See <<no-apm-data-found>> for information on how to fix this problem.
+*Ensure an APM data view exists*
+As a first step, you should ensure the correct data view exists.
+In {kib}, go to *Stack Management* > *Data views*.
+You should the APM data view--the default is
+`traces-apm*,apm-*,logs-apm*,apm-*,metrics-apm*,apm-*`.
+If you don't, the data view doesn't exist.
 
-Selecting the `apm-*` index pattern shows a listing of every field defined in the pattern.
+Navigate to the APM app in {kib} and select *Add data*.
+In the APM tutorial, click *Load Kibana objects* to create the APM data view.
+
+If creating an APM data view doesn't solve the problem,
+see <<no-apm-data-found>> for further troubleshooting.
 
 *Ensure a field is searchable*
 There are two things you can do to if you'd like to ensure a field is searchable:
 
 1. Index your additional data as {apm-guide-ref}/metadata.html[labels] instead.
 These are dynamic by default, which means they will be indexed and become searchable and aggregatable.
 
-2. Use the {apm-guide-ref}/configuration-template.html[`append_fields`] feature. As an example,
-adding the following to `apm-server.yml` will enable dynamic indexing for `http.request.cookies`:
-
-[source,yml]
-----
-setup.template.enabled: true
-setup.template.overwrite: true
-setup.template.append_fields:
-  - name: http.request.cookies
-    type: object
-    dynamic: true
-----
+2. Create a custom mapping for the field.
```

Comment:
I'm working on docs for editing the `@custom` component template mappings in https://github.com/elastic/apm-server/pull/6940.